stdimage/models.py

```diff
@@ -148,7 +156,8 @@ class StdImageField(ImageField):
     }
 
     def __init__(self, verbose_name=None, name=None, variations=None,
-                 force_min_size=False, *args, **kwargs):
+                 create_variations_manually=False, force_min_size=False,
```

Comment:
I still prefer a callable that is called when the image is saved to the storage.